buildflow/cli/main.py

```diff
@@ -333,28 +333,29 @@ def init(
         with open(requirements_txt_path, "w") as requirements_txt_file:
             requirements_txt_file.write(_DEFAULT_REQUIREMENTS_TXT)
 
+    os.makedirs(directory, exist_ok=True)
```

Comment:
This won't actually create the parent directory right? This is just going to essentially run `mkdir .` which is a noop?